alacritty/src/display/window.rs

```diff
@@ -379,6 +381,42 @@ impl Window {
         self.window().id()
     }
 
+    #[inline]
+    pub fn set_clipboard_content(&self, ty: ClipboardType, text: String) {
+        let mut mimes = HashSet::new();
+        mimes.insert(String::from("text/plain;charset=utf-8"));
+        mimes.insert(String::from("text/plain"));
+        mimes.insert(String::from("UTF8_STRING"));
```

Comment:
I feel like it might be convenient for winit to do this for me? Chances are these mimes are going to be **very** common.

Maybe either a `set_text` function or some helper to get the "normal text" mimes?

Reply:
Mime types are hard since you need a strategy to pick one of offered and provide your data with mime type you've offered. For example you can serve image and text at the same time. So you need a callback to decide how to represent your data.

Reply:
Yeah, winit could provide a helpers for mime type handling. Im just not sure how they should look.

Reply:
I just feel like the majority of clients will likely always write text only.